pkg/auth/oauth/external/handler_test.go

```diff
@@ -8,4 +8,10 @@ import (
 
 func TestHandler(t *testing.T) {
 	_ = handlers.NewUnionAuthenticationHandler(nil, map[string]handlers.AuthenticationRedirector{"handler": &Handler{}}, nil)
+
+	// Implement handlers.AuthenticationSuccessHandler
```

Comment:
Why?  If we want this behavior, why not create a type return from `DefaultState` that is a `State`, an `AuthenticationSuccessHandler`, and an `AuthenticationErrorHandler` and let the compiler do the work?